### ISSUE TITLE
[Bug] MacBook AIR M1 2020 controls for next/previous track are not working #140

### ISSUE THREAD

**Issue by Confucius75** [steve228uk/YouTube-Music#138]
When I am trying to control Music  on clicking  on F7 and F9 on my keyboard, to go to next/previous track, nothing happens.
Meanwhile,  tap on F8 to pause/resume  track is working.

Big Sur 11.3.1
YT Music Version 1.1.1 (10)
C02FM1WKQ6L4

**Comment by vibhavoswal** [comment]
Same issue with Big Sur on Intel Macbook

**Comment by mdallava** [comment]
I have the same issue my MacBook Pro with he OS "Catalina"

**Comment by nices0325** [comment]
me either

**Comment by duc-ios** [comment]
Works on my Macbook Air M1 by adding the app to System Preferences - Security & Privacy - Accessibility

**Comment by mdallava** [comment]
@duc-ios I cannot find it in Accessibility on my computer... did you do something special to see it there ?

**Comment by duc-ios** [comment]
@mdallava please check Privacy tab
https://i.stack.imgur.com/ARcRZ.png

**Comment by mdallava** [comment]
Yes sure I know where to find this tab but I was saying I can't find the application "YT Music" in Security & Privacy - Accessibility

**Comment by duc-ios** [comment]
@mdallava you need to tap the `+` button and add `YT Music` app

**Comment by mdallava** [comment]
Thanks, still not working for me, only the play/pause button is working...

**Comment by justAsascha** [comment]
For some reasons I needed to disable and enable it. But it works now. I think we can close this issue.

**Comment by fratule** [comment]
No, it's still not working on Mac Air M1. Even if we disable and enable again in Accessibility tab.

**Comment by maximemcduff** [comment]
I install with the website and is not working, i install with brew and its working if this can help someone.

**Comment by fratule** [comment]
@maximemcduff I will try and let you know. Thank you)

**Comment by fratule** [comment]
@maximemcduff You are right! When I installed it via brew every key is working 🤘

Thank you!

**Comment by fratule** [comment]
@steve228uk FYI

**Comment by unbeatable-101** [comment]
> @steve228uk FYI

@TimOliver Seems to be the one working on this project now

**Comment by jscheel** [comment]
I found that I had to fully remove the app from the list and re-add it, not just unchecking and rechecking the checkbox. I also had to restart the app. I wonder if this has something to do with the update process. I'm not sure what macOS uses to identify if the app is similar enough to still allow access to a specified app (of if it even uses any heuristics at all).

**Comment by kdschaffe** [comment]
No luck for me. I tried installing through Brew and enabling/disabling Accessibility (I did the process a couple times).

macOS Big Sur 11.6.1 
YT Music 1.0.6 (8)
